### PR TITLE
update highscores.js and style of the main page

### PR DESCRIPTION
**Pull request by alicjaguzik**

### CODE REVIEW

File: src/index.scss [hunk 1]
```diff
@@ -1,48 +1,66 @@
-.logo {
```
Comment (ruljin) [inline src/index.scss:1]: Styles for mobile version `360 x 640` are different than the one on Figma.

Comment (ruljin) [reply src/index.scss:1]: Same for `1440 x 1024` for the main page.

Comment (ruljin) [reply src/index.scss:1]: High scores for mobile version `360 x 640` are different than the one on Figma.

**Review by ruljin — CHANGES_REQUESTED** [review]

File: src/variables.scss [hunk 2]
```diff
@@ -20,6 +20,7 @@ $l-font-size-mobile: 42px;
 $xl-font-size-mobile: 92px;
 
 // breakpoints
+$s-breakpoint: 360px;
```
Comment (JustD1d1t) [inline src/variables.scss]: why we need variable with this resolution?
In my opinion we should start styling views from mobile to desktop (mobile first). I think that users on watches etc will not play this game.

**Pull request merged**